openapi.yaml

```diff
@@ -137,6 +205,42 @@ parameters:
       required:
         - data
       additionalProperties: false
+  financialAidPatchBody:
```

Comment:
Shouldnt this be required? It looks like the admissionsPatchBody isnt required either.

Reply:
Good catch, fixed.